package.json

```diff
@@ -44,29 +46,49 @@
     "release:minor": "node scripts/release/release.js minor",
     "release:patch": "node scripts/release/release.js patch",
     "test": "npm run jest && npm run compile && npm run nightwatch && npm run nightwatch:port-provided && wdio wdio.conf.js",
+    "pack": "./bin/webpack.js --config src/webpack/webpack.config.js",
```

Comment:
Should be able to simplified to: "pack": "webpack.js --config src/webpack/webpack.config.js",

Reply:
oops, i didn't mean to check this one in. it was just for testing.